pkg/scheduler/scheduler_test.go

```diff
@@ -38,7 +38,8 @@ import (
 	clientcache "k8s.io/client-go/tools/cache"
 	"k8s.io/client-go/tools/record"
 	"k8s.io/kubernetes/pkg/api/legacyscheme"
-	"k8s.io/kubernetes/pkg/controller/volume/persistentvolume"
+	_ "k8s.io/kubernetes/pkg/apis/core/install"
```

Comment:
Why I import this package explicitly is because we need register core API types (e.g. v1.Pod) into `legacyscheme.Scheme`. In old code, it is registered explicitly by `k8s.io/kubernetes/pkg/controller/volume/persistentvolume`.

BTW, another way is to use `k8s.io/client-go/kubernetes/scheme` which registers all API types by default.